crates/sui-indexer-builder/src/indexer_builder.rs

```diff
@@ -143,13 +161,13 @@ impl<P, D, M> Indexer<P, D, M> {
     {
         let tasks = self.storage.tasks(&self.name).await?;
         let backfill_tasks = tasks.backfill_tasks();
-        let latest_task = backfill_tasks.first();
+        let latest_backfill_task = backfill_tasks.first();
 
         // 1, create and update live task if needed
         if !self.disable_live_task {
             let from_checkpoint = max(
                 self.start_from_checkpoint,
-                latest_task
+                latest_backfill_task
```

Comment:
Isn't this the live task?

Reply:
`let latest_backfill_task = backfill_tasks.first();` gives you the backfill task with the highest target checkpoint.

This has nothing to do with live task. Are you asking why we are using a variable called `latest_backfill_task` in a if statement that seems to be related to live task?

Reply:
never mind. I was thinking the variable name was misnamed but seems appropriate. We assume the first backfill_task is the live task... makes sense.